.editorconfig

```diff
@@ -0,0 +1,12 @@
+# editorconfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Default settings:
+# A newline ending every file
+# Use 4 spaces as indentation
+[*]
```

Comment:
```suggestion
[*]
indent_style = space
indent_size = 2
```

Reply:
I was exploring this and unfortunately this setting doesn't play well with our indentation within comment blocks. If I can find a way to make that work, I will add it. Otherwise, this is already defined in the .clang-format file, so not too concerning.

Reply:
These settings don't work well and have unintended side effects.
Even outside of comments, it changes spacing on ternary operators and other if-blocks, causing misalignment, which we don't want:

![image](https://user-images.githubusercontent.com/6527137/94191187-03124300-fe62-11ea-8106-63feb3fb9aad.png)
![image](https://user-images.githubusercontent.com/6527137/94191260-17eed680-fe62-11ea-91c6-17431bb4271f.png)